addons/payment/static/src/js/payment_form.js

```diff
@@ -383,6 +383,7 @@ odoo.define('payment.payment_form', function (require) {
         // event handler when clicking on 'and more' to show more payment icon
         onClickMorePaymentIcon: function (ev) {
             ev.preventDefault();
+            ev.stopPropagation();
```

Comment:
why this change in this pr ? 

It is really needed, which behaviour is fixed by this line?

Reply:
@JKE-be , See the screenshot: https://drive.google.com/file/d/13hyGQ-D9hvOL4DmGUTVzSN5fIl_JsynE/view?usp=drivesdk

Right now, 'Ingenico' is selected, and we want to see more card icons of 'PayPal'. This should not select 'PayPal'. So we need to stop the propagation of the event bound on 'and more' button.